docs/writing-stories/introduction.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 'Introduction'
+---
+
+**A story captures the rendered state of a UI component**. It’s a function that returns a component’s state given a set of arguments. 
+
+Storybook uses the generic term arguments (args for short) when talking about React’s `props`, Vue’s `slots`, Angular’s `@input`, and other similar concepts.
+
+### Where to put stories
+
+A component’s stories are defined in a story file that lives alongside the component file. The story file is for development-only, it won't be included in your production bundle.
+
+```
+Button.js | ts
+Button.stories.js | ts
+```
+
+### Component Story Format
+
+We define stories according to the Component Story Format (CSF), an ES6 module-based standard that is portable between tools and easy to write. 
+
+The key ingredients are the **`default` export** that describes the component, and **named exports** that describe the stories.
+
+#### Default export
+
+The default export metadata controls how Storybook lists your stories and provides information used by addons. For example, here’s the default export for a story file `Button.stories.js`:
+
+```js
+// Button.stories.js
+
+import { Button } from './Button';
+
+export default {
+  title: ‘Components/Button’,
```

Comment:
```suggestion
  title: 'Components/Button',
```

Looks like some quotes have gotten messed up

Reply:
i've taken care of them in the get-started section and now here